examples/env.py

```diff
@@ -0,0 +1,29 @@
+"""
+This example showcases how Hera can dynamically set environmental variables
+"""
+
+from hera import ConfigMapEnvFromSpec, EnvSpec, Task, Workflow
+from hera.parameter import Parameter
```

Comment:
Minor, minor suggestion 
```suggestion
from hera import ConfigMapEnvFromSpec, EnvSpec, Task, Workflow, Parameter
```